src/WebJobs.Script/Binding/DocumentDBScriptBindingProvider.cs

```diff
@@ -43,7 +46,19 @@ public override bool TryCreate(ScriptBindingContext context, out ScriptBinding b
         /// <inheritdoc/>
         public override void Initialize()
         {
-            Config.UseDocumentDB();
+            DocumentDBConfiguration documentDBConfiguration = new DocumentDBConfiguration();
+
+            JObject configSection = (JObject)Metadata.GetValue("documentDB", StringComparison.OrdinalIgnoreCase);
```

Comment:
We were talking with Paul to do a renaming sometime in the future to CosmosDB in everything, but it would be a breaking change for current functions. 

I could not make the Lease Options work on the function level. Mainly because, even if I parse it correctly, I cannot pass it to the Attribute because it's not a basic Type, so it doesn't work in:

`[CosmosDBTrigger("ItemDb", "ItemCollection", LeaseOptions= new LeaseOptions(...))]`

That is why I put it on the global `host.json` level.